docs/table/index.rst

```diff
@@ -44,98 +39,93 @@ case, while the full `astropy.table` documentation is available from the
+  >>> import astropy.units as u
   >>> a = [1, 4, 5]
   >>> b = [2.0, 5.0, 8.2]
   >>> c = ['x', 'y', 'z']
-  >>> t = Table([a, b, c], names=('a', 'b', 'c'), meta={'name': 'first table'})
+  >>> t = QTable([a, b, c], names=('a', 'b', 'c'), meta={'name': 'first table'})
 
 If you have row-oriented input data such as a list of records, use the ``rows``
 keyword.  In this example we also explicitly set the data types for each column::
 
   >>> data_rows = [(1, 2.0, 'x'),
   ...              (4, 5.0, 'y'),
   ...              (5, 8.2, 'z')]
-  >>> t = Table(rows=data_rows, names=('a', 'b', 'c'), meta={'name': 'first table'},
-  ...           dtype=('i4', 'f8', 'S1'))
+  >>> t = QTable(rows=data_rows, names=('a', 'b', 'c'), meta={'name': 'first table'},
+  ...            dtype=('i4', 'f8', 'S1'))
 
-There are a few ways to examine the table.  You can get detailed information
-about the table values and column definitions as follows::
+You can now add a |Quantity| column with units in the table as shown below.  This
+stores a native |Quantity| within the table and brings the full power of
+:ref:`astropy-units` to the table.
 
-  >>> t  # doctest: +IGNORE_OUTPUT_3
-  <Table length=3>
-    a      b     c
-  int32 float64 str1
-  ----- ------- ----
-      1     2.0    x
-      4     5.0    y
-      5     8.2    z
+::
 
-You can also assign a unit to the columns. If any column has a unit
-assigned, all units would be shown as follows::
+  >>> t['d'] = [10, 20, 30] * u.m / u.s
 
-  >>> t['b'].unit = 's'
-  >>> t  # doctest: +IGNORE_OUTPUT_3
-  <Table length=3>
-    a      b       c
-           s
-  int32 float64 str1
-  ----- ------- ----
-      1     2.0    x
-      4     5.0    y
-      5     8.2    z
+If you prefer to not use |Quantity| (or the table has no data with units), then
+instead use the |Table| class to create tables.  See :ref:`quantity_and_qtable`
+for details on the differences and use cases.
 
-Finally, you can get summary information about the table as follows::
+There are a few ways to examine the table.  You can get detailed information
+about the table values and column definitions as follows::
 
-  >>> t.info  # doctest: +IGNORE_OUTPUT_3
-  <Table length=3>
-  name  dtype  unit
-  ---- ------- ----
-     a   int32
-     b float64    s
-     c    str1
+  >>> t
+  <QTable length=3>
+    a      b      c       d
+                        m / s
+  int32 float64 bytes1 float64
+  ----- ------- ------ -------
+      1     2.0      x    10.0
+      4     5.0      y    20.0
+      5     8.2      z    30.0
 
-A column with a unit works with and can be easily converted to an
-`~astropy.units.Quantity` object (but see :ref:`quantity_and_qtable` for
-a way to natively use `~astropy.units.Quantity` objects in tables)::
+Finally, you can get summary information about the table as follows::
 
-  >>> t['b'].quantity  # doctest: +FLOAT_CMP
-  <Quantity [2. , 5. , 8.2] s>
-  >>> t['b'].to('min')  # doctest: +FLOAT_CMP
```

Comment:
I think a unit conversion example is still useful here. Can you please add this back in, but in a way that works with your new example?

Reply:
OK.  It's just that people really should not be doing this.  These shims were put in place before Quantity mixins and they are just the wrong thing to do.

Reply:
I am not sure if I understand why people should not be doing this. If I have a column in seconds and I want to see the output in hours without changing the table content, what am I supposed to be doing?

Reply:
I think this is a good example right after creating the table with `b` - where you say "bring full power ... to the table". Just add a sentence: "Hence, all normal quantity operations work on this column. For instance::"

If we go this route, would it be an idea to make the add-a-column example use `Time`?

Reply:
Though you do `Time` later, so maybe just leave it as adding another `Quantity`

Reply:
> "Hence, all normal quantity operations work on this column. "

I was actually going in the opposite direction of stressing that normal quantity do NOT work, e.g. squaring a `Column` does not do anything to the `unit` and gives the wrong answer.  So the intent here was to purposely not highlight the (partway-there) quantity compatibility that is built-in to Column.  

What about moving documentation about this compatibility layer into the `Access table` section as a separate sub-section `Quantity-like methods in Column`  or some-such?

Reply:
> If I have a column in seconds and I want to see the output in hours without changing the table content, what am I supposed to be doing?

You should be using QTable and Quantity.  :smile:  But see my comment above about documenting all this separately, just not in `Getting Started`.

Reply:
@taldcroft - I think I managed to add my comment out of place: I meant that since you now create a `QTable` immediately, this is now the right place to say that any operation on the quantity column will behave just like for a regular quantity.